### DISCUSSION TITLE
Tipo de retorno na função declaracao do avaliador sintatico do visualg

### DISCUSSION THREAD

**Discussion by ItaloCobains** [DesignLiquido/delegua#156] [P&R] [answered]
### Tem alguma ideia de como tipar esse retorno da função declaracao ?

@leonelsanchesdasilva

![image](https://user-images.githubusercontent.com/83548040/200037652-6cebd036-188e-4428-baa4-887f275300cd.png)

**Comment by leonelsanchesdasilva** [comment, marked as answer]
Eu faria `Declaracao | Declaracao[] | Construto | Construto[] | any`, porque tem o retorno nulo ali na quebra de linha.
